### DISCUSSION TITLE
AdGuard and AdGuard VPN

### DISCUSSION THREAD

**Discussion by tychoregter** [elrumo/macOS_Big_Sur_icons_replacements#1128] [Icon request]
I would love to see new icons for AdGuard and AdGuard VPN

[AdGuard](https://adguard.com/en/adguard-mac/overview.html)
[AdGuard VPN (beta)](https://adguard-vpn.com/en/beta.html)

**Comment by johnlamox** [comment]
A little bit late, but done one for them yesterday.
Get it here: https://macosicons.com/u/johnlamox
